src/app/ubs-admin/components/ubs-client-bonuses/ubs-client-bonuses.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="bonuses-container">
+  <div class="no-bonuses-wrapper" *ngIf="!totalBonuses && !isLoading">
+    <span class="no-bonuses-text">Вам ще не нараховано бонусів.</span>
+    <a href="#" class="more-about-bonuses-link">Дізнатися за що нараховуються бонуси</a>
+  </div>
+
+  <div class="bonuses-exist-wrapper" *ngIf="totalBonuses && !isLoading">
+    <div class="total-bonuses-number">
+      <p class="total-bonuses-number__header">Бонуси</p>
+      <p class="total-bonuses-number__amount">{{ totalBonuses }}</p>
+    </div>
+
+    <a href="" class="more-about-bonuses-link link-center">Про систему бонусів</a>
+
+    <div class="bonuses-table-wrapper">
+      <table mat-table [dataSource]="dataSource" class="table" matSort>
+        <ng-container matColumnDef="dateOfEnrollment">
+          <th mat-header-cell *matHeaderCellDef class="table-header" mat-sort-header>Дата зарахування</th>
```

Comment:
Add either an 'id' or a 'scope' attribute to this <th> tag.

Reply:
done